### ISSUE TITLE
Show the bar by clicking on the tab bar

### ISSUE THREAD

**Issue by jmain33** [Quicksaver/The-Fox--Only-Better#208]
Use a single left click on the tab bar's empty space to immediately show the omni-bar (or whatever it's actually called).
I use multi-rowed tabs by way of TabMixPlus, so there's always plenty of empty space, and single clicks on the tab bar aren't really utilized by anything else (in any way that I know of); this enhancement would go very well with that type of setup.

**Comment by Quicksaver** [comment]
I don't understand, to click on the tab bar, you'd have to move the mouse over it, and by doing this the toolbars should be fully visible by then. Are the toolbars not appearing when you move your mouse over your tabs?

**Comment by jmain33** [comment]
There is a delay configured for when over the tabs bar (or upper region in general).  This suggestion is just a method to immediately show it by mouse.  I already make use of the F6 key as a keyboard function.
In the short time I've spent getting my feet wet with Beyond Australis, this came forward as the most convenient way to show it upon request.  I'm satisfied with the delay I've assigned in the options dialog (I'm just constantly clicking that empty space expecting it to appear on demand :expressionless:).

**Comment by Quicksaver** [comment]
Ah ok, so mouse movement is working at least.

I don't believe this is possible at all though, because that Firefox makes that empty tabs area work in a sort of non-existing way. For example, I already have to trick it into registering mouse movement correctly there (it doesn't actually register any mouse movement in that area). Clicks should also be possible of course, but when clicking it you aren't actually focusing anything, so it would have to respond to the clicks directly, which I would like to avoid because I believe there are add-ons (like Tab Mix Plus) that already add their own click handlers there.

I may give it a look, but I won't make any promises on this one. I hope you understand.

**Comment by jmain33** [comment]
Oh, I understand completely; I'm sure working these types of things out is a tall order.  Thanks for looking into it, as I'd really appreciate such a feature.

**Comment by jmain33** [comment]
In the event that my original idea proves too difficult to implement, I've thought of a couple of alternatives: 

- Remove the requirement that the cursor be still in order for the bar to show.
- Make clicking on the thin strip cause the bar to display.

**Comment by Quicksaver** [comment]
> Remove the requirement that the cursor be still in order for the bar to show.

It doesn't? What do you mean by this, it definitely should show the toolbars by **moving** the cursor over the top area, not only by placing it there and leaving it there.

**Comment by jmain33** [comment]
Well, that's how it's working on my machine :confounded: ?
I'm using multi-row tabs (4 rows) by way of tab mix plus, so there is plenty of <ins>activation</ins> area.  The bar only shows once the cursor has stayed still for a ≈_second_.

**Comment by Quicksaver** [comment]
Well, yeah... If you set the delay to a higher value, then it will keep "resetting" if you keep moving the mouse around, otherwise its sensitivity would be pretty much nulled (as in it would make no sense to have that setting at all).

If you want it to appear faster, lower the sensitivity in the preferences. After all, that's what it's supposed to do.

**Comment by Quicksaver** [comment]
(corrected the wording in the post above to be less confusing)

**Comment by jmain33** [comment]
OK, I thought the sensitivity worked as a simple delay.  I don't know why having it reset while the mouse is in the activation area is sensible (or maybe I should say why it functions that way). If it's in the activation area (and has not left) the timer should simply count to its set value, regardless of how fast my mouse moves.

**Comment by Quicksaver** [comment]
Actually, the fact the countdown resets is technically a side-effect of how everything is implemented, to get around it would be extremely laborious to be completely honest, and it's something I **really** don't want to do. 😜 

But it was the desired behavior from the start. Those sliders are named "sensitivity" and not "delays" for a reason. Simple case: say in those four rows of tabs you're looking for a specific one, and move the mouse with your eyes (it's an involuntary action that almost everyone does). When that happens, you probably don't care about showing the toolbars at all. You also probably don't care if you're moving the mouse trying to aim at a specific tab or button and it takes anything more than a reflex.

In short: "delay" is not the same as "sensitivity", and I truly believe you're underestimating how good for you that distinction ends up being. 😄

**Comment by jmain33** [comment]
:laughing: I figured from your previous response that the "sensitivity" was somehow ingrained to work the way it does…  I'm re-tweaking that value to something that responds quicker (it was at 900, now set to 300).

Still holding out hope for some form of _instant_ activation to become evidently doable.  I know that Aha! moment could take an unknowable amount of time to happen, until then I'm plenty happy with your work.
